runelite-client/src/main/java/net/runelite/client/plugins/envisionplugins/breakhandler/BreakHandlerPlugin.java

```diff
@@ -0,0 +1,101 @@
+package net.runelite.client.plugins.envisionplugins.breakhandler;
+
+import com.google.inject.Provides;
+import lombok.extern.slf4j.Slf4j;
+import net.runelite.client.config.ConfigManager;
+import net.runelite.client.plugins.Plugin;
+import net.runelite.client.plugins.PluginDescriptor;
+import net.runelite.client.plugins.envisionplugins.breakhandler.enums.BreakHandlerStates;
+import net.runelite.client.plugins.envisionplugins.breakhandler.ui.enums.TimeDurationType;
+import net.runelite.client.ui.ClientToolbar;
+import net.runelite.client.ui.NavigationButton;
+import net.runelite.client.util.ImageUtil;
+
+import javax.inject.Inject;
+import java.awt.image.BufferedImage;
+
+@PluginDescriptor(
+        name = PluginDescriptor.Envision + "Break Handler",
+        description = "Break Handler for Microbot",
+        tags = {"microbot", "utility"},
+        enabledByDefault = true
```

Comment:
not sure if this is needed here, but do we want the BreakHandlerPlugin to always be on? 

if yes, than you can use the         alwaysOn = true  property